doc/build-windows.md

```diff
@@ -68,7 +72,7 @@ Then build using:
     cd ..
     ./autogen.sh # not required when building from tarball
     CONFIG_SITE=$PWD/depends/x86_64-w64-mingw32/share/config.site ./configure --prefix=/
-    make
+    make -j4
```

Comment:
Ff you're going to add `-j` in places, at least explain that it impacts parallellism in a comment.